site/app/views/fr/view.json

```diff
@@ -1,6 +1,6 @@
 {
-  "title": "Quickstarts French",
-  "description": "Follow along with our French tutorials and step-by-step walkthroughs to get you up and running with Postman and APIs.",
+  "title": "Quickstarts Postman",
```

Comment:
Can you revert to `Quickstarts French` this title also informs the language switcher, so it makes sense when on the main page, to switch over to "Quickstarts French" or something similar with the word "French".